src/pretix/control/templates/pretixcontrol/organizers/edit.html

```diff
@@ -200,6 +200,19 @@ <h1>
                         {% bootstrap_field sform.cookie_consent_dialog_button_yes layout="control" %}
                         {% bootstrap_field sform.cookie_consent_dialog_button_no layout="control" %}
                     </fieldset>
+                    <fieldset id="tab-organizer-accessibility">
+                        <legend>{% trans "Accessibility" %}</legend>
+                        <div class="alert alert-legal">
+                            {% blocktrans trimmed with a_href="https://docs.pretix.eu/trust/accessibility/" %}
+                                Some jurisdictions, including the European Union, require you to publish information
+                                about the accessibility of your ticket shop. You can find a template in
+                                <a href="https://docs.pretix.eu/trust/accessibility/" target="_blank">our documentation</a>.
+                            {% endblocktrans %}
+                        </div>
+                        {% bootstrap_field sform.accessibility_url layout="control" %}
+                        {% bootstrap_field sform.accessibility_title layout="control" %}
+                        {% bootstrap_field sform.accessibility_text layout="control" %}
```

Comment:
I’d remove the help-text and move it before the input, IMHO having it below the input does not help here.
```suggestion
                        <div class="row">
                            <div class="col-md-9 col-md-offset-3">
                                <p class="text-muted">{% trans "Instead of an URL, you can also configure a text that will be shown within pretix. This will be ignored if a URL is configured." %}</p>
                            </div>
                        </div>
                        {% bootstrap_field sform.accessibility_text layout="control" %}
```